test/testca/keys.go

```diff
@@ -0,0 +1 @@
+package testca
```

Comment:
Is this file a placeholder for future code/keys? or should we remove it?

Reply:
Left over from copying this from the go-spiffe library :) I'll remove.